icechunk-python/python/icechunk/__init__.py

```diff
@@ -22,13 +22,13 @@
 from zarr.core.sync import SyncMixin
 
 __all__ = [
-    "__version__",
     "IcechunkStore",
     "S3Credentials",
     "SnapshotMetadata",
     "StorageConfig",
     "StoreConfig",
     "VirtualRefConfig",
+    "__version__",
```

Comment:
@paraseba This was the fix you needed. IDK why teh fix wasnt automatically applied or detected locally